monix-catnap/jvm/src/test/scala/monix/catnap/MVarJVMSuite.scala

```diff
@@ -175,7 +134,7 @@ abstract class BaseMVarJVMSuite(parallelism: Int) extends TestSuite[SchedulerSer
     }
   }
 
-  test("MVar (concurrent) — issue #380: with foreverM; without latch") { implicit ec =>
+  test("MVar  — issue #380: with foreverM; without latch") { implicit ec =>
```

Comment:
uh, I derped and left double spaces as I was removing the concurrent/async distinction 😅